lib/util.js

```diff
@@ -291,6 +306,63 @@ exports.getVideoDescription = function(html) {
 
 
 /**
+ * Get video Owner from html.
+ *
+ * @param {String} body
+ * @return {Object}
+ */
+var authorRegexp = /<a href="\/channel\/([\w-]+)"[^>]+>(.+?(?=<\/a>))/
+var aliasRegExp = /<a href="\/user\/([^"]+)/
+exports.getAuthor = function(body) {
+  var ownerinfo = this.between(body, '<div id="watch7-user-header" class=" spf-link ">', '<div id="watch8-action-buttons" class="watch-action-buttons clearfix">');
+  ownerinfo = new Entities().decode(ownerinfo)
+  if (ownerinfo === '') {
+    return {};
+  }
+  var match = ownerinfo.match(authorRegexp);
+  return {
+    ref: '/channel/'+match[1],
+    id: match[1],
+    name: match[2],
+    avatar: this.between(ownerinfo, 'data-thumb="', '"'),
+    user: (ownerinfo.match('/user/') ? '/user/'+ownerinfo.match(aliasRegExp)[1] : 'none'),
```

Comment:
Spaces around `+`

The parenthesis around this ternary statement are not needed here, and it looks it's as clear without it too.

Should use `null` so that no user doesn't point to https://www.youtube.com/user/none. If given `null`, someone can more easily check for that. Also, how often does the user not get matched? Maybe a test could be added where it finds the channel but no user?